From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kjartan Einarsson <kjartanreinarsson@gmail.com>
Date: Mon, 19 Jan 2026 12:00:00 +0000
Subject: [PATCH] chore: minimal android config for bcsc dev

Essential changes only - namespace, applicationId, package declarations,
Firebase config, and version numbers. No cosmetic changes (icons, app name).

Signed-off-by: Kjartan Einarsson <kjartanreinarsson@gmail.com>
---
 app/android/app/build.gradle                       |  6 +++---
 app/android/app/google-services.json               | 17 +++++++++--------
 app/android/app/src/main/AndroidManifest.xml       |  2 +-
 .../java/ca/bc/gov/BCWallet/MainActivity.kt        |  2 +-
 .../java/ca/bc/gov/BCWallet/MainApplication.kt     |  2 +-
 5 files changed, 16 insertions(+), 15 deletions(-)

diff --git a/app/android/app/build.gradle b/app/android/app/build.gradle
index f552ceb7..12914cd9 100644
--- a/app/android/app/build.gradle
+++ b/app/android/app/build.gradle
@@ -97,7 +97,7 @@ def jscFlavor = 'org.webkit:android-jsc:+'
 def nativeArchitectures = project.getProperties().get("reactNativeDebugArchitectures")
 
 android {
-    namespace 'ca.bc.gov.BCWallet'
+    namespace 'ca.bc.gov.id.servicescard.dev'
     ndkVersion rootProject.ext.ndkVersion
     compileSdkVersion rootProject.ext.compileSdkVersion
     buildToolsVersion rootProject.ext.buildToolsVersion
@@ -116,11 +116,11 @@ android {
     }
 
     defaultConfig {
-        applicationId "ca.bc.gov.BCWallet"
+        applicationId "ca.bc.gov.id.servicescard.dev"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode System.getenv("VERSION_CODE") ? System.getenv("VERSION_CODE").toInteger() : 1
-        versionName System.getenv("VERSION_NAME") ? System.getenv("VERSION_NAME").toString() : "0.0.1"
+        versionCode System.getenv("VERSION_CODE") ? System.getenv("VERSION_CODE").toInteger() : 3654
+        versionName System.getenv("VERSION_NAME") ? System.getenv("VERSION_NAME").toString() : "4.0.0"
 
         missingDimensionStrategy 'react-native-camera', 'general'
         ndk {
diff --git a/app/android/app/google-services.json b/app/android/app/google-services.json
index a3a58790..8643db07 100644
--- a/app/android/app/google-services.json
+++ b/app/android/app/google-services.json
@@ -1,33 +1,34 @@
 {
   "project_info": {
-    "project_number": "1008044298583",
-    "project_id": "bc-wallet-mobile",
-    "storage_bucket": "bc-wallet-mobile.appspot.com"
+    "project_number": "209333430748",
+    "firebase_url": "https://bcsc-app-test-fcm.firebaseio.com",
+    "project_id": "bcsc-app-test-fcm",
+    "storage_bucket": "bcsc-app-test-fcm.firebasestorage.app"
   },
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "1:1008044298583:android:56bf7c25b882152a44ae12",
+        "mobilesdk_app_id": "1:209333430748:android:1c4641260a97bfe5",
         "android_client_info": {
-          "package_name": "ca.bc.gov.BCWallet"
+          "package_name": "ca.bc.gov.id.servicescard.dev"
         }
       },
       "oauth_client": [
         {
-          "client_id": "1008044298583-12lpkisi403oa5bc5hs0800o48cl4nhj.apps.googleusercontent.com",
+          "client_id": "209333430748-f89kks1sr4an87kmkilim6e1jj2gd0nc.apps.googleusercontent.com",
           "client_type": 3
         }
       ],
       "api_key": [
         {
-          "current_key": "AIzaSyB9TMILh7AQmEGCdwvvmDeN-WG4G345p9I"
+          "current_key": "AIzaSyBUm3mvXOZ_EGrRnyVFNgJIQJZQhGR2_uw"
         }
       ],
       "services": {
         "appinvite_service": {
           "other_platform_oauth_client": [
             {
-              "client_id": "1008044298583-12lpkisi403oa5bc5hs0800o48cl4nhj.apps.googleusercontent.com",
+              "client_id": "209333430748-f89kks1sr4an87kmkilim6e1jj2gd0nc.apps.googleusercontent.com",
               "client_type": 3
             }
           ]
diff --git a/app/android/app/src/main/AndroidManifest.xml b/app/android/app/src/main/AndroidManifest.xml
index cd4c656c..dc895efe 100644
--- a/app/android/app/src/main/AndroidManifest.xml
+++ b/app/android/app/src/main/AndroidManifest.xml
@@ -1,4 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="ca.bc.gov.BCWallet"
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="ca.bc.gov.id.servicescard.dev"
   xmlns:tools="http://schemas.android.com/tools">
   <uses-feature android:name="android.hardware.camera" />
   <uses-feature android:name="android.hardware.camera.autofocus" />
diff --git a/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainActivity.kt b/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainActivity.kt
index 1234567..abcdefg 100644
--- a/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainActivity.kt
+++ b/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainActivity.kt
@@ -1,4 +1,4 @@
-package ca.bc.gov.BCWallet
+package ca.bc.gov.id.servicescard.dev
 
 import android.content.Intent
 import android.content.res.Configuration
diff --git a/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainApplication.kt b/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainApplication.kt
index 1234567..abcdefg 100644
--- a/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainApplication.kt
+++ b/app/android/app/src/main/java/ca/bc/gov/BCWallet/MainApplication.kt
@@ -1,4 +1,4 @@
-package ca.bc.gov.BCWallet
+package ca.bc.gov.id.servicescard.dev
 
 import android.app.Application
 import com.facebook.react.PackageList
